portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -643,19 +643,6 @@
         <url-pattern>/oranalysis.do</url-pattern>
     </servlet-mapping>
 
```

Comment:
we should make sure that we update frontend to compatible version (meaning it can handle ln route)

Reply:
updated it